templates/step_function/consignment_export_definition.json.tpl

```diff
@@ -0,0 +1,132 @@
+{
+  "Comment": "A state machine to run the Fargate task to export the consignment",
+  "StartAt": "Run ECS task",
+  "States": {
+    "Run ECS task": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::ecs:runTask.waitForTaskToken",
+      "HeartbeatSeconds": 60,
+      "TimeoutSeconds": 1800,
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "States.HeartbeatTimeout",
+            "States.Timeout"
+          ],
+          "MaxAttempts": 3
```

Comment:
Should the `MaxAttempts` be a parameter?

Reply:
This one makes sense as a parameter as well, I've changed it.